lib/src/utilities.dart

```diff
@@ -70,3 +70,18 @@ int bitCount(int i) {
   i += i >> 16;
   return i & 0x0000003F;
 }
+
+bool isPowerOf2(int value) => value != 0 && (value & (value - 1)) == 0;
+
+const String radixDigits = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+void reverseList(List<Object?> values) {
+  var half = values.length ~/ 2;
+  var last = values.length - 1;
+  for (var i = 0; i < half; i++) {
+    var tmp = values[i];
+    var fromEnd = last - i;
+    values[i] = values[fromEnd];
+    values[fromEnd] = tmp;
+  }
```

Comment:
I didn't notice where this is used.

I find it easier to understand the indexing when written like this:
```
int start = 0, end = values.length-1;
while (start < end) {
  final first = values[start];
  final last = values[end];
  values[start++] = last;
  values[end--] = first;
}
```
This also generalizes nicely for a reverse-subrange operation.

Reply:
Good catch, I stopped using it again.